dateutil/parser/__init__.py

```diff
@@ -0,0 +1,8 @@
+# -*- coding: utf-8 -*-
+from ._parser import parse, parser, parserinfo
+from ._parser import DEFAULTPARSER, DEFAULTTZPARSER
+from .isoparser import Isoparser, isoparse
+
+__all__ = ['parse', 'parser', 'parserinfo',
+           'isoparse', 'Isoparser',
+           'InvalidDatetimeError', 'InvalidDateError', 'InvalidTimeError']
```

Comment:
You need to bring this to this namespace as well, don't you? It is neither a module nor a local variable.

I think doing this will break `from dateutil.parser import *`

Reply:
You might want to add some tests to verify current behaviour on the ` import * ` and general imports before moving it :X

Reply:
Good catch! I'll add that and add some tests.